micrometer-core/src/main/java/io/micrometer/core/instrument/binder/tomcat/TomcatMetrics.java

```diff
@@ -72,4 +101,145 @@ public void bindTo(MeterRegistry reg) {
             .tags(tags)
             .register(reg);
     }
+
+
+    private void waitForTomcatJmxRegistration() {
+        MBeanServer server = mBeanServerProvider.get();
+        long endWait = System.currentTimeMillis() + maxRegistrationWait.toMillis();
+
+        //noinspection InfiniteLoopStatement
+        while (System.currentTimeMillis() < endWait) {
+            try {
+                if(!server.queryNames(new ObjectName("Tomcat:type=GlobalRequestProcessor,*"), null).isEmpty()){
+                    return;
+                }
+
+                Thread.sleep(500);
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
Nit for me to address.

Reply:
Would you like it to be a no-op or use SLF4J?